Fraudio/index.ts

```diff
@@ -0,0 +1,48 @@
+import * as selectively from "selectively"
+import * as authly from "authly"
+
+export interface Fraudio {
+	score: number
+	recommendation: "green" | "yellow" | "red"
+	id: authly.Identifier
+	threshold: number
+	model: string
+	notes?: string
+	transaction?: string
+}
+
+export namespace Fraudio {
+	export function is(value: Fraudio | any): value is Fraudio {
+		return (
+			typeof value == "object" &&
```

Comment:
Implicit not null check:
```suggestion
			typeof value == "object" && value &&
```

Reply:
Fixed